README.md

```diff
@@ -82,64 +42,84 @@ module "teleport_bootstrap_script" {
-* [`project`]: String(optional): Project where this node belongs to, will be the second part of the node name. Defaults to ''
-* [`teleport_version`]: String(optional): Teleport version you want to install. Defaults to 2.4.0
-* [`tokens`]: List(optional): List of tokens you want to add to the authentication server. Defaults to []
-* [`cw_logs_cpu`]: Integer(optional): The number of CPU units used by the CloudWatch logs task. It can be expressed as an integer using CPU units, for example 1024, or as a string using vCPUs, for example 1 vCPU or 1 vcpu, in a task definition but will be converted to an integer indicating the CPU units when the task definition is registered. Defaults to 128
-* [`cw_logs_memory`]: Integer(optional): The amount of memory (in MiB) used by the CloudWatch logs task. It can be expressed as an integer using MiB, for example 1024, or as a string using GB, for example 1GB or 1 GB, in a task definition but will be converted to an integer indicating the MiB when the task definition is registered. Defaults to 128
-* [`cw_logs_memory_reservation`]: Integer(optional): The soft limit (in MiB) of memory to reserve for the CloudWatch logs container. When system memory is under contention, Docker attempts to keep the container memory to this soft limit; however, your container can consume more memory when it needs to, up to either the hard limit specified with the memory parameter (if applicable), or all of the available memory on the container instance, whichever comes first. Defaults to 64
-* [`elb_subnets`]: List(required): Subnets where to deploy the Teleport ELB
-* [`web_ssl_certificate_arn`]: String(optional): ARN of the SSL certificate to use for the Teleport ELB
-* [`web_allowed_cidr_blocks`]: List(optional): CIDR blocks that are allowed to access the web interface of the proxy server. Defaults to `["0.0.0.0/0"]`
-* [`cli_allowed_cidr_blocks`]: List(optional): CIDR blocks that are allowed to access the cli interface of the proxy server. Defaults to `["0.0.0.0/0"]`
-* [`tunnel_allowed_cidr_blocks`]: List(optional): CIDR blocks that are allowed to access the reverse tunnel interface of the proxy server. Defaults to `["0.0.0.0/0"]`
-* [`ecs_instances_sg_id`]: String(required): Security group ID of the backend ECS instances running Teleport
-* [`teleport_log_severity`]: String(optional): Logging configuration for Teleport, possible severity values are `INFO`, `WARN` and `ERROR`. Defaults to `ERROR`
-* [`create_dns_record`]: String(optional): Create DNS records to reach teleport. Defaults to `"true"`
-
-### Output
-/
+
+* [`teleport_proxy_sg_id`]: String(required): Security group id of the `proxy` server.
+* [`teleport_node_sg_id`]: String(required): Security group id of the `node` server.
+* [`teleport_auth_sg_id`]: String(required): Security group id of the `auth` server.
```

Comment:
Can you add a note about the fact that the proxy and auth are in the same server at the moment or this is not true? In the second case I am a bit confused